src/main/java/no/sikt/sws/WorkspaceStripper.java

```diff
@@ -49,19 +54,37 @@ public static String prefixBody(String body, String workspace, String index) {
     }
 
 
-    public static String prefixBody(List<JsonNode> gatewayBody, String workspaceprefix) {
+    public static String prefixIndexesBulkBody(List<JsonNode> gatewayBody, String workspacePrefix) {
+        if (gatewayBody == null || workspacePrefix == null) {
+            return null;
```

Comment:
Do we need to log what was missing before returning null? Or would it be obvious if something fails?

Reply:
-> Exception with message